samples/sensor/tcs3472x/README.rst

```diff
@@ -0,0 +1,99 @@
+.. _tcs3472x:
+
+TCS3472x Color Sensor
```

Comment:
Please look at making this driver generic for any color sensor. See `samples/sensor/accel_polling` for example

Reply:
Hi, do your mean that i should rewrite the sample or the driver itself?

Reply:
I like your idea of rewriting the sample, maybe i could test it with an additional APDS9960 from Adafruit or so. Wouldn't it be better that this MR should only contain the driver component then?

Reply:
Sorry for the confusing typo in my earlier comment. I meant to say that the sample application should be generic, not the driver itself. It used to be the case that we had multiple accelerometer samples, multiple magnetometer samples, multiple temperature sensor samples, etc. and it got out of control having a new sample added for every new sensor driver, so we have been trying to simplify down to one sample per type of sensor.

If you happen to have another color sensor on hand to test the generic sample that would be great, but not required.